test/integration/tags/include_tag_test.rb

```diff
@@ -235,4 +235,11 @@ def test_including_via_variable_value
 
     assert_template_result "Product: Draft 151cm ", "{% assign page = 'product' %}{% include page for foo %}", "foo" => { 'title' => 'Draft 151cm' }
   end
+
+  def test_including_with_strict_variables
+    template = Liquid::Template.parse("{% include 'simple' %}", error_mode: :warn)
```

Comment:
No, it does not need to exists to show the issue under test. In fact, AFAIR I added the template but removed it again before comitting, since it was not needed here.